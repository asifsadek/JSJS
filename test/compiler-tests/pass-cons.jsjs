val x = (/\(x) => x;)([1,2]) :: [[1,2,3],[4,5]];

print(x);
