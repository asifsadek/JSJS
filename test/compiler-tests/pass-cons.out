List [ List [ 1, 2 ], List [ 1, 2, 3 ], List [ 4, 5 ] ]
